CBattle/package/cog.py

```diff
@@ -114,11 +122,20 @@ async def add(self, interaction: discord.Interaction["BallsDexBot"], countryball
             return
 
         battle_player.balls.append(BattleBall.from_ballinstance(countryball))
-        await interaction.response.send_message(f"{countryball.countryball.country} added!", ephemeral=True)
+        emj = self.bot.get_emoji(countryball.countryball.emoji_id)
+        await interaction.response.send_message(f"`#{countryball.id}` {emj} {countryball.countryball.country} removed!", ephemeral=True)
         await battle.accept_view.update()
 
     @app_commands.command()
     async def remove(self, interaction: discord.Interaction["BallsDexBot"], countryball: BallInstanceTransform):
+        """
+        Removes a ball from the battle..
```

Comment:
```suggestion
        Removes a countryball from the battle.
```
`ball` should be `countryball` as the name won't be translated for custom collectible names.